iOS_SDK/OneSignalSDK/Source/OSInAppMessage.m

```diff
@@ -195,13 +195,15 @@ - (void)encodeWithCoder:(NSCoder *)encoder {
     [encoder encodeObject:_messageId forKey:@"messageId"];
     [encoder encodeObject:_variants forKey:@"variants"];
     [encoder encodeObject:_triggers forKey:@"triggers"];
+    [encoder encodeObject:_displayStats forKey:@"redisplay"];
 }
 
 - (id)initWithCoder:(NSCoder *)decoder {
     if (self = [super init]) {
         _messageId = [decoder decodeObjectForKey:@"messageId"];
         _variants = [decoder decodeObjectForKey:@"variants"];
         _triggers = [decoder decodeObjectForKey:@"triggers"];
+        _displayStats = [decoder decodeObjectForKey:@"redisplay"];
```

Comment:
Use the variable name as the key
`_displayStats = [decoder decodeObjectForKey:@"displayStats"];`

Reply:
good catch!